tutorials/2d/2d_transforms.rst

```diff
@@ -1,84 +1,182 @@
+    :ref:`scale <class_Transform2D_method_get_scale>`.
+
+- **Window transform**
+    In order to scale and position the *Window's* content as described in
+    :ref:`doc_multiple_resolutions`, each :ref:`Window <class_Window>` contains a *window transform*.
+    It is for example responsible for the black bars at the *Window's* sides so that the `Viewport`
+    is displayed with a fixed aspect ratio.
+
+- **Window position**
+    Every *Window* also has a :ref:`position <class_Window_property_position>` to describe its
+    position within its embedder.
+
+- **SubViewportContainer shrink transform**
+    :ref:`stretch <class_SubViewportContainer_property_stretch>` together with
+    :ref:`stretch_shrink <class_SubViewportContainer_property_stretch_shrink>` declare for a
+    *SubViewportContaner* if and by what integer factor the contained *SubViewport* should be
+    scaled in comparison to the container's size.
 
 Transform functions
 -------------------
 
-Obtaining each transform can be achieved with the following functions:
+The above graphic shows all available transform functions (exposed to GDScript and also
+internal). All transforms are directed from right to left, this means multiplying a transform with
+a coordinate results in a coordinate system further to the left, multiplying the
+:ref:`affine inverse <class_Transform2D_method_affine_inverse>` of a transform results in a
+coordinate system further to the right:
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    canvas_pos = get_global_transform() * local_pos
+    local_pos = get_global_transform().affine_inverse() * canvas_pos
+
+ .. code-tab:: csharp
 
-+----------------------------------+---------------------------------------------------------------------------------------------+
-| Type                             | Transform                                                                                   |
-+==================================+=============================================================================================+
-| CanvasItem                       | :ref:`CanvasItem.get_global_transform() <class_CanvasItem_method_get_global_transform>`     |
-+----------------------------------+---------------------------------------------------------------------------------------------+
-| CanvasLayer                      | :ref:`CanvasItem.get_canvas_transform() <class_CanvasItem_method_get_canvas_transform>`     |
-+----------------------------------+---------------------------------------------------------------------------------------------+
-| CanvasLayer+GlobalCanvas+Stretch | :ref:`CanvasItem.get_viewport_transform() <class_CanvasItem_method_get_viewport_transform>` |
-+----------------------------------+---------------------------------------------------------------------------------------------+
+    canvasPos = GetGlobalTransform().Xform(localPos)
+    localPos = GetGlobalTransform().AffineInverse().Xform(canvasPos)
```

Comment:
Transform2D also has a `*` operator in C#, the `Xform` method was removed.

```suggestion
    canvasPos = GetGlobalTransform() * localPos;
    localPos = GetGlobalTransform().AffineInverse() * canvasPos;
```

Reply:
Thank you for the comments. I will include your improvements in the PR.